apis/tempo/v1alpha1/tempostack_types.go

```diff
@@ -689,6 +689,12 @@ type JaegerQueryMonitor struct {
 	// +kubebuilder:validation:Optional
 	// +operator-sdk:csv:customresourcedefinitions:type=spec,displayName="Prometheus endpoint"
 	PrometheusEndpoint string `json:"prometheusEndpoint"`
+	// REDMetricsNamespace defines the a prefix used retrieve span rate, error, and duration (RED) metrics.
+	// By default no namespace is set.
```

Comment:
How does this default play out with collector 0.109.0? 
By default the collector 0.109.0 uses the namespace `traces.span.metrics` .


Can customers set an empty namespace in the collector 0.109.0? Was it tested? 

If we default to an empty namespace it implies customers have to set the empty namespace in the collector. It seems better if the new version bundle (collector and tempo) do not require any additional configuration.

Reply:
Maybe we should flip it to a boolean enabling and disabling the prefix. Since its not configurable on the collector.  (A configurable prefix instead of the featuregate would probably have been the best way to change the default namespace on the collector...)

In that case flipping the default would be the breaking change and we can discuss when this action should take place.